recipes/src/recipes/RecipeDetail.test.js

```diff
@@ -0,0 +1,65 @@
+import { act, render, fireEvent, screen } from '@testing-library/react';
+
+import RecipeDetail from './RecipeDetail';
+import testData from '../../data/testData';
+
+const { recipe1, recipe2, recipe3 } = testData;
+
+describe('RecipeDetail', () => {
+
+  const recipes = [ recipe1, recipe2, recipe3 ];
+  const selected = 2;
+
+  test('it should display the recipe detail', () => {
+    const mockSetUpdating = jest.fn();
+    const mockDeleteReipe = jest.fn();
+
+    render(<RecipeDetail recipes={recipes}
+                         selected={selected}
+                         setUpdating={mockSetUpdating}
+                         deleteRecipe={mockDeleteReipe}/>);
+
+    expect(screen.getByText(recipe2.name)).not.toBeNull();
```

Comment:
better to be assertive `expect(screen.getByText(recipe2.name)).toBeInTheDocument()`

Reply:
Yes - that's better, thanks.
I didn't spot it in the docs.